lib/src/sfu_connection.dart

```diff
@@ -14,15 +24,20 @@ class SfuConnection {
   final Function(int trackId, RTCTrackEvent event) onTrack;
 
   /// Creates a new [SfuConnection].
-  ///
-  /// Provide [_localStream] if this is an outgoing connection sending the stream's audio and video to the SFU;
-  /// otherwise, this is an incoming connection receiving audio from the SFU.
-  SfuConnection(this.trackId, this.onConnectionState, this.onIceCandidate, this.onSdpOffer, this.onTrack,
-      [MediaStream? localStream])
-      : _localStream = localStream;
+  SfuConnection({
+    required this.direction,
+    required this.onConnectionState,
+    required this.onIceCandidate,
+    required this.onSdpOffer,
+    required this.onTrack,
+    required this.trackId,
+  });
```

Comment:
it would have been nice to take this occasion to send the constructor as the first defined element of the class.